test/functionalTest/cases/1697_more_than_one_bad_input/more_than_one_bad_input.test

```diff
@@ -80,41 +80,40 @@ echo
 01. Provoke FOUR Bad Inputs
 ===========================
 HTTP/1.1 400 Bad Request
-Content-Length: 97
-Content-Type: application/json
-Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Content-Type: application/json
+Content-Length: 97
 
 {
     "description": "Service not found. Check your URL as probably it is wrong.",
     "error": "BadRequest"
 }
 HTTP/1.1 400 Bad Request
-Content-Length: 75
-Content-Type: application/json
-Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Content-Type: application/json
+Content-Length: 75
 
 {
     "description": "Errors found in incoming JSON buffer",
     "error": "ParseError"
 }
 HTTP/1.1 405 Method Not Allowed
-Content-Length: 63
-Content-Type: application/json
```

Comment:
This header should be removed.

Note this test is failing, see https://github.com/telefonicaid/fiware-orion/actions/runs/4919583948/jobs/8876244677

```
----------- Failing tests ------------------
  o  1492_limit_zero/limit_zero_types.test
  o  1697_more_than_one_bad_input/more_than_one_bad_input.test
  o  1916_fiware_correlator/fiware_correlator_forward_and_notifications.test
```